CHANGES.md

```diff
@@ -1,3 +1,23 @@
+## edge-19.5.2
+
+* CLI
+  * Fixed `linkerd check` and `linkerd dashboard` failing when any control plane
+    pod is not ready, even when multiple replicas exist (as in HA mode)
+* Controller
+  * Fixed control plane components failing on startup when the Kubernetes API
+    returns an `ErrGroupDiscoveryFailed`
+* Proxy
+  * Added a dispatch timeout that bounds the amount of time a request can be
+    buffered in the proxy
+  * Removed the limit on the number of concurrently active service discovery
+    to the Destination service
+* Internal
+  * Fixed potentially flaky assertions in the service profile integration tests
```

Comment:
Question - is "potentially flaky" going to be understood by everyone? Would "Improved assertions in the service profile integration tests to avoid potential instability" be better? TIOLI